core/src/main/java/net/sourceforge/jnlp/Parser.java

```diff
@@ -502,11 +502,12 @@ private JREDesc getJRE(final XmlNode node) throws ParseException {
             vmArgs = null;
         }
         final String vendor = getAttribute(node, JREDesc.VENDOR_ATTRIBUTE, null);
+        final boolean require32Bit = "true".equals(getAttribute(node, JREDesc.REQUIRE_32_BIT_ATTRIBUTE, "false"));
```

Comment:
Can you use `equalsIgnoreCase` or `Boolean.parseBoolean()`

Reply:
done